apt2aptly/src/main.rs

```diff
@@ -339,6 +339,8 @@ async fn sync_dist(
                     &publish::PublishOptions {
                         distribution: Some(dist_path),
                         signing: Some(signing),
+                        skip_bz2: true,
+                        skip_contents: true,
```

Comment:
This really should be a command line option ; but that can come later

Reply:
Will add as something to customize later